packages/medusa/src/commands/utils/get-migrations.js

```diff
@@ -1,98 +1,6 @@
 import glob from "glob"
-import { isString } from "lodash"
-import {
-  createRequireFromPath,
-  getConfigFile,
-  isDefined,
-} from "medusa-core-utils"
-import path from "path"
-import { handleConfigError } from "../../loaders/config"
-import { MEDUSA_PROJECT_NAME } from "../../loaders/plugins"
-
-function createFileContentHash(path, files) {
-  return path + files
-}
-
-// TODO: Create unique id for each plugin
-function createPluginId(name) {
-  return name
-}
-
-/**
- * Finds the correct path for the plugin. If it is a local plugin it will be
- * found in the plugins folder. Otherwise we will look for the plugin in the
- * installed npm packages.
- * @param {string} pluginName - the name of the plugin to find. Should match
- *    the name of the folder where the plugin is contained.
- * @return {object} the plugin details
- */
-function resolvePlugin(pluginName) {
-  // Only find plugins when we're not given an absolute path
-  if (!existsSync(pluginName)) {
-    // Find the plugin in the local plugins folder
-    const resolvedPath = path.resolve(`./plugins/${pluginName}`)
-
-    if (existsSync(resolvedPath)) {
-      if (existsSync(`${resolvedPath}/package.json`)) {
-        const packageJSON = JSON.parse(
-          fs.readFileSync(`${resolvedPath}/package.json`, `utf-8`)
-        )
-        const name = packageJSON.name || pluginName
-        // warnOnIncompatiblePeerDependency(name, packageJSON)
-
-        return {
-          resolve: resolvedPath,
-          name,
-          id: createPluginId(name),
-          options: {},
-          version:
-            packageJSON.version || createFileContentHash(resolvedPath, `**`),
-        }
-      } else {
-        // Make package.json a requirement for local plugins too
-        throw new Error(`Plugin ${pluginName} requires a package.json file`)
-      }
-    }
-  }
-
-  const rootDir = path.resolve(".")
-
-  /**
-   *  Here we have an absolute path to an internal plugin, or a name of a module
-   *  which should be located in node_modules.
-   */
-  try {
-    const requireSource =
-      rootDir !== null
-        ? createRequireFromPath(`${rootDir}/:internal:`)
-        : require
-
-    // If the path is absolute, resolve the directory of the internal plugin,
-    // otherwise resolve the directory containing the package.json
-    const resolvedPath = path.dirname(
-      requireSource.resolve(`${pluginName}/package.json`)
-    )
-
-    const packageJSON = JSON.parse(
-      fs.readFileSync(`${resolvedPath}/package.json`, `utf-8`)
-    )
-    // warnOnIncompatiblePeerDependency(packageJSON.name, packageJSON)
-
-    return {
-      resolve: resolvedPath,
-      id: createPluginId(packageJSON.name),
-      name: packageJSON.name,
-      version: packageJSON.version,
-    }
-  } catch (err) {
-    throw new Error(
-      `Unable to find plugin "${pluginName}". Perhaps you need to install its package?`
-    )
-  }
-}
+import { isDefined } from "medusa-core-utils"
```

Comment:
todo: from the utils package

Reply:
@adrien2p there are still some small cleanups left, I will go through everything one more time and clean up any orphan code, it was just difficult to do it all in 1 go and have it reviewable 😄

Reply:
just doubled check and yes you are right :D